.circleci/config.yml

```diff
@@ -1,16 +1,23 @@
-version: 2.1
+version: 2
```

Comment:
circleci CLI didn't want to help me with any 2.1 stuff, so just dropped this to 2 for now